src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services1-1.php

```diff
@@ -7,8 +7,8 @@
 use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
 use Symfony\Component\DependencyInjection\Exception\LogicException;
 use Symfony\Component\DependencyInjection\Exception\RuntimeException;
-use Symfony\Component\DependencyInjection\ServiceLocator;
 use Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag;
+use Symfony\Component\DependencyInjection\ServiceLocator;
```

Comment:
Why? This is expected, I've updated the PhpDumper to do so (https://github.com/symfony/symfony/pull/21600/files#diff-f7b23d463cba27ac5e4cb677f2be7623R902).
I know it doesn't matter much, but this use directive was added in master, so I thought we could change this order to respect alphabetical one as soon as it's not released.